spec/namespaces/ml.yaml

```diff
@@ -735,6 +758,26 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/ml.get_stats@200'
+  /_plugins/_ml/tools:
+    get:
+      operationId: ml.get_tools.0
+      x-operation-group: ml.get_tools
+      x-version-added: '2.12'
+      description: Get tools.
+      responses:
+        '200':
+          $ref: '#/components/responses/ml.get_tools@200'
+  /_plugins/_ml/tools/{tool_name}:
+    get:
+      operationId: ml.get_tools.1
+      x-operation-group: ml.get_tools
+      x-version-added: '2.12'
+      description: Get tools.
+      parameters:
+        - $ref: '#/components/parameters/ml.get_tools::path.tool_name'
+      responses:
+        '200':
+          $ref: '#/components/responses/ml.get_tools@200'
```

Comment:
Given that these return different formatted response bodies, I think these should have been two different operation groups, ie. `get_all_tools` and `get_tool`.